node/plugin/cmd/main.go

```diff
@@ -97,8 +99,24 @@ func pluginOps(ctx *cli.Context) {
 		return
 	}
 
+	host, dispersalPort, retrievalPort, err := core.ParseOperatorSocket(config.Socket)
+	if err != nil {
+		log.Printf("Error: failed to parse operator socket: %v", err)
+		return
+	}
+
+	socket := string(core.MakeOperatorSocket(host, dispersalPort, retrievalPort))
```

Comment:
It looks you are doing some back and forth (parsing and then making again). Can it just use `config.Socket`?